cli/cmd/build/config/config.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	apicore "github.com/agntcy/dir/api/core/v1alpha1"
+)
+
+type Locator struct {
+	Type string `yaml:"type"`
+	URL  string `yaml:"url"`
+}
+
+type Config struct {
```

Comment:
todo: fully expand this struct based on all config fields without mapping standard types to needed structs via methods